signup.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>FITBIT</title>
+
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous" />
+
+  <link rel="stylesheet"
+    href="https://deploy-preview-10--suspicious-sammet-33f1b3.netlify.app/Components/components.css" />
+  <link rel="stylesheet" href="https://suspicious-sammet-33f1b3.netlify.app/Components/components.css" />
+  <link rel="stylesheet" href="common.css" />
+  <link rel="stylesheet" href="signup.css" />
+</head>
+
+<body class="wrapper">
+  <!--  ******************************** Navbar Start ********************************  -->
+  <nav class="navbar">
+    <div>
+      <img alt="hamburger-icon" src="images/hamburger.svg" class="hamburger-icon" id="hamburger-icon" />
+    </div>
+    <div class="logo-container">
+      <div class="logo">
+        <img class="logo-image" alt="logo-image" src="images/neatUI.svg" />
+        <span class="logo-text">FiT</span>
+        BiT
+      </div>
+    </div>
+    <div class="nav-items-container-end">
+      <div class="login-search-bar-container">
+        <a class="button button-primary" href="login.html">Login</a>
+      </div>
+    </div>
+  </nav>
+  <!--  ******************************** Navbar End ********************************  -->
+
+  <!--  ******************************** SignUp Container Start ********************************  -->
+  <div class="spacer-3"></div>
+  <main>
+    <img src="images/blob.svg" class="blob blob1" />
```

Comment:
use alt attribute as it is a good practice !!

Reply:
Done